examples/emberjs/todomvc/app/components/todo-list.js

```diff
@@ -5,7 +5,7 @@ import Component from '@ember/component';
 export default Component.extend({
```

Comment:
Should we be using es classes?

Reply:
Updated the PR description with more information.